secret-readme.md

```diff
@@ -0,0 +1,147 @@
+# Earthly Secret Store
+
+The secret store can be used to share secrets between team members or across multiple computers and CI systems.
+
+This service is currently experimental.
+
+# Usage
+
+Please upgrade to the latest version, at the time of writing this guide, that version is v0.3.18
+
+## Registering for an account
+
+First, you must register for an account:
+
+    earth account register --email <email>
+
+An email will be sent to you containing a verification token, next run:
+
+    earth account register --email <email> --token <token>
+
+This command will prompt you to set a password, and to optionally register a public-key for password-less authentication.
+
+## Login / Logout
+
+It is recommended that you register a public rsa key during registration; if this is done, you will be logged in automatically whenever
+earth needs to authenticate you. If you did not supply a public key, then your plain-text password will be cached on your local disk under
+`~/.earthly/auth-token`, which will be used to log you in. If this file is deleted, you will need to run `earth account login` to re-create it.
+
+To logout, you can run `earth account logout`, which deletes the `~/.earthly/auth-token` file from your disk.
+
+## Interacting with the private user secret store from the command line
+
+Each user will have a non-sharable private userspace which can be referenced by `/user/...`; this can be though of as your home directory.
```

Comment:
```suggestion
Each user will have a non-sharable private userspace which can be referenced by `/user/...`; this can be thought of as your home directory.
```